Examples/Run/Reconstruction/Common/RecCKFTracks.cpp

```diff
@@ -32,9 +32,8 @@
 using namespace Acts::UnitLiterals;
 using namespace ActsExamples;
 
-int main(int argc, char* argv[]) {
-  GenericDetector detector;
-
+int recCKFTracks(int argc, char* argv[],
```

Comment:
```suggestion
int runRecCKFTracks(int argc, char* argv[],
```
Suggestion: naming similar to e.g. the `runFatras` main method

Reply:
Thanks. Changed.